index.html

```diff
@@ -19,5 +19,37 @@ <h1>Experience the rush today</h1>
                 </div>
             </div>
         </section>
+        <section class = "sect_content">
+            <div class = "container">
+                <div class="feature">
+                        <div class="feature__icon_container">
+                            <img src="./img/Timer.png" alt="timer" class="feature__icon">
+                        </div class="feature__icon_container">
+                    <h6>Designer guff</h6>
+                    <p class="feature__desc">Minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum </p>
+                </div>
+                <div class="feature">
+                        <div class="feature__icon_container">
+                            <img src="./img/Dashboard.png" alt="dashboard" class="feature__icon">

```

Comment:
[Optional] The `alt` attribute is used for accessibility as well so we should describe in few words what is on the picture. I see a schematic representation of speedometer so `alt="speedometer icon"` looks better for me. I don't know how to explain to a blind person how `dashboard` might look